### PR TITLE
Removing Guava and introducing java 8

### PR DESCRIPTION
**Pull request by jontejj**

### CODE REVIEW

**Comment by dizzzz** [comment]
@adamretter .... looks good?

File: common-test/pom.xml [hunk 1]
```diff
@@ -20,6 +20,12 @@
 			<artifactId>guava-testlib</artifactId>
 			<version>${guava.version}</version>
 		</dependency>
+		<dependency>
+			<!-- The stuff that java should have had from the start -->
```
Comment (dizzzz) [inline common-test/pom.xml:24]: the title is about removing guava ... still required? or only in test?

Comment (jontejj) [reply common-test/pom.xml:24]: It's only used by tests. The common-test module should never be depended on by production code anyway, so guava will not follow along when jargo is packaged.

Comment (dizzzz) [reply common-test/pom.xml:24]: thnx!

**Comment by adamretter** [comment]
@jontejj Will this ever get merged and release?

**Comment by jontejj** [comment]
Will try to push it and deploy to maven central during this week!

**Comment by adamretter** [comment]
@jontejj Awesome. Thank you :-)

**Comment by jontejj** [comment]
Merged it. Will deploy to maven central later tonight at home!

**Comment by jontejj** [comment]
Version 0.4.2 has now been deployed and is available on maven central! Enjoy :)

**Comment by adamretter** [comment]
Fantastic Thank you @jontejj :-)

**Pull request merged**